store/tikv/txn.go

```diff
@@ -268,6 +272,7 @@ func (txn *tikvTxn) Commit(ctx context.Context) error {
 	// pessimistic transaction should also bypass latch.
 	if txn.store.txnLatches == nil || txn.IsPessimistic() {
 		err = committer.execute(ctx)
+		txn.onCommitted(err)
```

Comment:
We could ignore the inner transactions whose connection id is zero in the `committer` since we will not query them.

Reply:
Will inner transaction use client's session?

Reply:
It will use the inner session with zero connection id.